devmaster/themes/boots/template.php

```diff
@@ -656,10 +656,11 @@ function boots_preprocess_page(&$vars){
     }
 
     // Task nodes only have project nid and environment name.
-    if (is_numeric($vars['node']->project)) {
+    if (is_numeric($vars['node']->project ?? NULL)) {
```

Comment:
Wrong position of closing `)`

`is_numeric($vars['node']->project) ?? NULL`
not
`is_numeric($vars['node']->project ?? NULL)`